webapp/lib/charts.dart

```diff
@@ -154,7 +154,7 @@ class DailyTimeseriesLineChartView {
     chart = chartjs.Chart(canvas.getContext('2d'), chartConfig);
   }
 
-  void updateChart(List<Map<DateTime, num>> updatedCountsAtTimestampList, {String timeScaleUnit = 'day', num upperLimit}) {
+  void updateChart(List<Map<DateTime, num>> updatedCountsAtTimestampList, {String timeScaleUnit = 'day', num upperLimit, DateTime xUpperLimit}) {
```

Comment:
Can you add and set the `xLowerLimit` as well as you did for the system event charts?

Reply:
Yes, I've added it